src/applications/vaos/appointment-list/components/AppointmentsPageV2/index.jsx

```diff
@@ -59,8 +59,9 @@ export default function AppointmentsPageV2() {
   const location = useLocation();
   const [hasTypeChanged, setHasTypeChanged] = useState(false);
   const showScheduleButton = useSelector(state => selectFeatureRequests(state));
-  const isWelcomeModalDismissed = useSelector(state =>
-    selectIsWelcomeModalDismissed(state),
+  const isWelcomeModalDismissed = useSelector(
+    state => selectIsWelcomeModalDismissed(state),
+    shallowEqual,
```

Comment:
Actually, I don't believe `shallowEqual` is needed here since we expect just a boolean value from this selector. It's needed for the `getPastAppointmentListInfo` selector in the `PastAppointmentsListNew` component.